rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.7.2-release-notes.adoc

```diff
@@ -0,0 +1,59 @@
+= RPA Builder 6.7.2 Release Notes
+
+RPA Builder is the on-premises application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== September 27, 2024
+ 
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+
+* The `StNetSh.exe` tool, part of the RPA Builder setup, now includes a digital signature to prevent security warnings. 
+* The Google Chrome driver is updated to version 130. 
+* The MS Edge driver is updated to version 129.
+
+// == Compatibility
+include::partial$release-notes/rn-headings.adoc[tag=compatibility,leveloffset=+1]
+
+[%header%autowidth.spread]
+|===
+|Application |Version
+|RPA Recorder | 6.6.0 and later
+|RPA Bot | 1.0.0 and later
+|===
+
+// == Fixed Issues
+
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]
+
+[%header,cols="5a,1a"]
+|===
+| Issue Resolution | ID
+| Execute Excel Macro no longer shows an incorrect error message when trying to include it inside an Excel Session. | W-16603539
+| Run Program no longer truncates the command line to 1024 characters. | W-16721834
+| Read Email no longer fails to download and save mail attachments. | W-16723360
+| The logfile no longer shows entire array values to improve readability. | W-16760451
+| RPA Builder no longer causes an error when using Json Query inside a loop. | W-16766643
```

Comment:
```suggestion
| RPA Builder no longer causes an error when using JSON Query inside a loop. | W-16766643
```

Reply:
The name of the action step is Json Query